### PR TITLE
Fixed issue #2989

### LINKED ISSUE #2989: Unable to navigate to rated skill from dashboard.
**Actual Behaviour**

Currently it's unable navigate to a skill found under the my ratings section.

**Expected Behaviour**

The user should be able to navigate to the selected skill without any issue.

**Screenshots of the issue**

UnableToNavigate

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by anishagg17**
earlier it was unable navigate to a skill found under the my ratings section due to some errors while allocating link .
Now it has been resolved

Fixes #2989

### CODE REVIEW

**Comment by anishagg17** [comment]
Issue is resolved when i visit https://pr-3045-fossasia-susi-web-chat.surge.sh

**Comment by AuraOfDivinity** [comment]
@anishagg17 It occurs for a niche set of skills. For ex:- Try Rating the ApiTests skill and using the link.

File: src/components/cms/Dashboard/MyRatings.js [hunk 1]
```diff
@@ -93,19 +95,26 @@ class MyRatings extends Component {
...
-                    ratingTimestamp,
+                    group,
+                    language,
                     skillStar,
+                    ratingTimestamp,
                   } = skill;
+                  let v = '';
+                  for (let i = 0; i < skillName.length; i++) {
+                    let character = skillName.charAt(i);
+                    if (character === character.toUpperCase() && i !== 0) {
```
[the remark this answers is not in the document]
Comment (anishagg17) [reply src/components/cms/Dashboard/MyRatings.js]: sir,

For a required skill to display previously link displayed was:
https://susi.ai/Games,%20Trivia%20and%20Accessories/drawacard/language

after my fix link points to the right directory:
https://susi.ai/Games,%20Trivia%20and%20Accessories/draw_a_card/en

Clearly you can observe the difference between the two links
I surely would  put this skill name transformation into a separate util.

[the remark this answers is not in the document]
Comment (anishagg17) [reply src/components/cms/Dashboard/MyRatings.js]: I have uploaded transformation util at
https://github.com/fossasia/susi.ai/pull/3062

**Pull request closed without merging**